config/rbac/role.yaml

```diff
@@ -1,4 +1,35 @@
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
Is the cluster role required also when the operator is installed in a single namespace?

Reply:
Yes. The tokenreview granted to the broker SA is a cluster resource and requires a cluster role.